htdocs/js/jsx/incompletes_candidates_panel.jsx

```diff
@@ -0,0 +1,380 @@
+                }.bind(this)
+            }.bind(this)
+        }
+    },
+    getNumPages: function() {
+        var numPages = Math.floor(this.props.table_rows.length / this.state.pageSize);
+        if (this.props.table_rows.length % this.state.pageSize > 0){
+            numPages++
+        }
+        return numPages
+    },
+    handlePageChange: function(pageNum) {
+        this.setState({currentPage: pageNum})
+    },
+    render:function(){
+        var page = this.getPage();
+        var rows_to_map = page.table_rows;
+        var children_to_map = this.props.children;
+        var rows_for_current_page = rows_to_map.map(function(row){
+            var mapped = React.Children.map(children_to_map, function (child) {
+                return React.cloneElement(child, {
+                    row: row
+                });
+            });
+            return mapped;
+        });
+        if(rows_for_current_page.length){
+            var table_contents = <table className="table table-hover table-primary table-bordered colm-freeze">
+                <PagedRowHeader header_row={this.props.table_headers}/>
+                <tbody>
+                {rows_for_current_page}
+                </tbody>
+            </table>
+
+        }
+        else{
+            var table_contents = "There is no data to display";
+        }
+        return <div>
+        {table_contents}
+            <nav>
+		{pager(page)}
+            </nav>
+        </div>
+    }
+})
+
+
+/**
+ * Renders a pager component.
+ */
+function pager(page) {
+    var pageLinks = []
+    if (page.currentPage > 1) {
+        pageLinks.push(<li onClick={page.handleClick(page.currentPage - 1)}><span>‹</span></li>)
+        if (page.currentPage > 2) {
+            pageLinks.push(<li onClick={page.handleClick(1)}><span>1</span></li>)
+            pageLinks.push(<li><span>...</span></li>)
+        }
+    }
+    if(page.numPages > 1){  pageLinks.push(<li className="active"><span>{page.currentPage}</span></li>)
+    }
+    if (page.currentPage < page.numPages) {
+        pageLinks.push(<li onClick={page.handleClick(page.currentPage + 1)}><span>{page.currentPage + 1}</span></li>)
+        if (page.currentPage < page.numPages - 1) {
+            pageLinks.push(<li onClick={page.handleClick(page.currentPage + 2)}><span>{page.currentPage + 2}</span></li>)
+        }
+        if(page.currentPage < page.numPages - 2){
+            pageLinks.push(<li onClick={page.handleClick(page.currentPage + 3)}><span>{page.currentPage + 3}</span></li>)
+        }
+        if(page.currentPage < page.numPages - 3){
+            pageLinks.push(<li className="disabled"><span>...</span></li>)
+            pageLinks.push(<li onClick={page.handleClick(page.numPages)}><span>{page.numPages}</span></li>)
+        }
+        pageLinks.push(<li onClick={page.handleClick(page.currentPage + 1)}><span aria-hidden="true">›</span>
+        </li>)
+    }
+    return <ul className="pagination pagination-sm">{pageLinks}</ul>
```

Comment:
No as it doesn't actually link to different pages. It just creates a paged table (you cannot link to specific pages in the table).